packages/cli/src/cli.ts

```diff
@@ -39,6 +41,7 @@ export async function cli(args: string[] = process.argv) {
     )
     .demandCommand()
     .middleware(logVersionMiddleware, true)
+    .middleware(logConfigFileMiddleware, true)
```

Comment:
Is it desirable to run before validation with `true` for this middleware? If we wanted exactly the same behaviour as check I believe it runs with false

Reply:
I'm noticing this is run multiple times is this because it runs for each sub command as well? We may want to follow the existing version middleware which only logs on first run